app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -429,9 +428,8 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
-  // TODO (#1855): Resolve ktlint max line in app module test
   @Test
-  fun testAudioWithCellular_openRatioExploration_clickAudioIcon_changeConfiguration_checkOpensCellularAudioDialog() { // ktlint-disable max-line-length
+  fun testAudioCell_ratioExploration_audioIcon_configChange_opensCellularAudioDialog() {
```

Comment:
Suggestion:
Instead of `Cell` use `Cellular` and as an adjustment we can use `ratioExp`.
`testAudioCellular_ratioExp_audioIcon_configChange_opensCellularAudioDialog `

Reply:
Done